lib/pakyow/app/assets/console/styles/console.scss

```diff
@@ -334,6 +340,38 @@ aside {
     color: darken($color-background, 25%);
     margin-top: 0;
   }
+
+  @include mobile {
+    width: 100%;
+    padding: 5px 0;
+    height: 25px;
+    top: 0;
+    right: 0;
+    margin-left: 0 !important;
+    display: inline;
+    left: 0% !important;
+    margin-top: -5px;
+
+    nav {
+      background-color: $color-xgray;
+      padding: 10px 0;
+      margin-bottom: 10px !important;
+      border-bottom: 1px solid $color-border;
+      overflow-x: auto;
+    }
+
+    ul {
+      width: 90%;
+      margin: auto;
+      text-align: center;
+      overflow-x: auto;
```

Comment:
I can't get rid of the text-align center because everything will be shifted to the left and won't be even in menu. 

<img width="404" alt="screen shot 2017-02-09 at 8 12 01 am" src="https://cloud.githubusercontent.com/assets/8171067/22786769/834a6e2a-ee9f-11e6-8055-24c1648253e5.png">